Debian-BTS: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=661215
Author: Steve Langasek <steve.langasek@ubuntu.com>
Description: Don't check ENABLE_LDAP twice
 This source is already being conditionally compiled based on ENABLE_LDAP
 being set in the Makefile.  All this extra check does is cause umich_ldap.so
 to always be compiled as an *EMPTY* object because nothing sets ENABLE_LDAP
 before the #ifdef!
Bug-Ubuntu: https://bugs.launchpad.net/bugs/939232

diff -up a/umich_ldap.c b/umich_ldap.c
--- a/umich_ldap.c	2011-12-06 07:28:10.000000000 +1100
+++ b/umich_ldap.c	2012-02-27 13:21:34.000000000 +1100
@@ -32,8 +32,6 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifdef ENABLE_LDAP
-
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <netdb.h>
@@ -1302,4 +1300,3 @@ struct trans_func *libnfsidmap_plugin_in
 {
 	return (&umichldap_trans);
 }
-#endif
